introduction.js

```diff
@@ -1 +1,12 @@
 // Your code goes here.
+var name = "Mark Eldridge";
+var age1 = 27;
```

Comment:
Hey, Mac! I noticed that you still had a failing test, and it looks like it's because this variable is called `age1` instead of `age`.